test/integration/features/basic.feature

```diff
@@ -17,6 +17,15 @@ Feature: Basic test
             """
             Use "crc [command] --help" for more information about a command.
             """
+
+    @darwin @linux @windows
+    Scenario: CRC status
+        When executing "crc status" fails
+        Then stderr should contain 
+        """
+        [Error while getting cluster status: Machine "crc" does not exist. Use "crc start" to add a new one.]
```

Comment:
@jsliacan don't put exact error message just have a substring of it `Machine "crc" does not exist. Use "crc start" to add a new one` because this will mostly be consistent but exact error might change.